src/gemdat/path.py

```diff
@@ -205,6 +205,101 @@ def free_energy_graph(F: np.ndarray,
                                'dijkstra-exp']
 
 
+def calculate_path_difference(path_sites1: list, path_sites2: list) -> float:
+    """Calculate the difference between two paths. This difference is defined
+    as the percentage of sites that are not shared between the two paths.
+
+    Parameters
+    ----------
+    path_sites1 : list
+        List of sites defining the first path
+    path_sites2 : list
+        List of sites defining the second path
+
+    Returns
+    -------
+    difference : float
+        Difference between the two paths
+    """
+
+    # Find the shortest and longest paths
+    shortest_path = path_sites1 if len(path_sites1) <= len(
+        path_sites2) else path_sites2
+    longest_path = path_sites2 if shortest_path is path_sites1 else path_sites1
```

Comment:
There must be a cleverer way to do this, but just keep it simple for readability 😅

```suggestion
    if len(path1) <= len(path2):
        shortest, longest = path1, path2
    else:
        shortest, longest = path2, path1
```